src/applications/disability-benefits/all-claims/pages/newDisabilityFollowUp.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { createSelector } from 'reselect';
+import dateUI from 'us-forms-system/lib/js/definitions/date';
+
+import { getNewDisabilityName } from '../utils';
+import disabilityLabels from '../content/disabilityLabels';
+
+import fullSchema from '../config/schema';
+
+const {
+  cause,
+  primaryDisability,
+  mistreatmentDescription,
+  disabilityStartDate
+} = fullSchema.properties.newDisabilities.items.properties;
+
+export const disabilityNameTitle = ({ formData }) => {
+  return (
+    <legend className="schemaform-block-title schemaform-title-underline">{getNewDisabilityName(formData.diagnosticCode)}</legend>
+  );
+};
+
+export const uiSchema = {
+  newDisabilities: {
+    items: {
+      'ui:title': disabilityNameTitle,
+      cause: {
+        'ui:title': '',
+        'ui:widget': 'radio',
+        'ui:options': {
+          labels: {
+            NEW: 'My disability was caused by—or got worse because of—an injury or exposure during my service in the military.',
+            SECONDARY: 'My disability was caused by another disability (for example, I have a limp that caused a bad back).',
+            VA: 'My disability was caused by VA mistreatment.'
+          },
+          updateSchema: (formData, causeSchema, causeUISchema, index) => {
+            return {
+              title: `Please tell us what caused your ${disabilityLabels[formData.newDisabilities[index].diagnosticCode]}`
+            };
+          }
+        }
+      },
+      primaryDisability: {
+        'ui:title': 'Which disability caused the disability you’re claiming here?',
+        'ui:required': (formData, index) => formData.newDisabilities[index].cause === 'SECONDARY',
+        'ui:options': {
+          expandUnder: 'cause',
+          expandUnderCondition: 'SECONDARY',
+          updateSchema: createSelector(
```

Comment:
This is building the enum list based on the rated disabilities and the new ones that have been added (aside from the current one), based on Alex's guidance.

Reply:
Is the expectation that there will always be a `ratedDisability` in this situation?

Reply:
Yeah, I think hiding it when there are no possible options is a good choice. My understanding is that this list should be all the rated disabilities, plus any new disabilities other than the current one.